src/lib/comms/sol-socket-dtls-impl-tinydtls.c

```diff
@@ -371,7 +352,7 @@ call_user_read_cb(struct dtls_context_t *ctx, session_t *session, uint8_t *buf,
         return -EINVAL;
     }
 
-    if (socket->read.cb(socket->read.data, socket))
+    if (socket->read.cb(socket->data, socket))
```

Comment:
i've tried not changed things that were not necessary. But I'll change it before commit